econml/validate/results.py

```diff
@@ -205,18 +209,38 @@ def plot_uplift(self, tmt: Any):
             raise ValueError(f'Invalid treatment; must be one of {self.treatments[1:]}')
 
         df = self.curves[tmt].copy()
-        df['95_err'] = 1.96 * df['err']
+
+        if err_type is None:
+            df['95_err'] = 1.96 * df['err']
+        elif err_type == 'ucb2':
+            df['95_err'] = df['uniform_critical_value'] * df['err']
+        elif err_type == 'ucb1':
+            df['95_err'] = df['uniform_one_side_critical_value'] * df['err']
+        else:
+            raise ValueError("Invalid error type; must be one of [None, 'ucb2', 'ucb1']")
```

Comment:
```suggestion
            raise ValueError(f"Invalid error type {err_type}; must be one of [None, 'ucb2', 'ucb1']")
```

Reply:
(Note that this will require a corresponding change to your test where you check the error message)